.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,14 @@
+{
+    "image": "netdata/devenv", 
+    "extensions":[
+		"golang.go",
+		"exiasr.hadolint",
+		"ms-python.python",
+		"timonwong.shellcheck",
+		"redhat.vscode-yaml",
+		"dbaeumer.vscode-eslint",
+		"jasonnutter.search-node-modules",
+		"mgmcdermott.vscode-language-babel",		
```

Comment:
Do we need all those extension to develop in golang?

Reply:
You are correct. Removing them.

Reply:
Actually, we have a single devenv for all of Netdata, so I would argue that the overhead is minimal.  I would prefer to have the same support for every repo.

Reply:
So if you want to add an extensions for python you are going to update all the repositories?

Reply:
That's the idea. What do you think @ilyam8 ? Perhaps @Ferroin could give his view as well. I just want the best possible contributor experience (less friction as possible)